packages/help-center/src/contexts/HelpCenterContext.tsx

```diff
@@ -66,15 +86,56 @@ const defaultContext: HelpCenterRequiredInformation = {
 
 const HelpCenterRequiredContext = createContext< HelpCenterRequiredInformation >( defaultContext );
 
+function useNewInteractionsBotConfig() {
+	const experimentName = 'wpcom_help_center_ai_workflow_and_prompt_changes';
+	const query = useQuery( {
+		queryKey: [ 'new-interactions-bot-slug', experimentName ],
+		queryFn: () =>
+			canAccessWpcomApis()
+				? wpcomRequest< { variations: Record< typeof experimentName, keyof typeof botSlugMap > } >(
+						{
+							path: '/experiments/0.1.0/assignments/wpcom',
+							apiNamespace: 'wpcom/v2',
+							query: {
+								experiment_name: experimentName,
+							},
+						}
+				  )
+				: apiFetch< { variations: Record< typeof experimentName, keyof typeof botSlugMap > } >( {
+						path: addQueryArgs( 'jetpack/v4/explat/assignments', {
+							experiment_name: experimentName,
+							platform: 'wpcom',
+							as_connected_user: 'true',
+						} ),
+				  } ),
+	} );
+
+	if ( query.data?.variations && experimentName in query.data.variations ) {
+		// null -> control
+		const variant = query.data.variations[ experimentName ] ?? 'control';
+		const botSlug = botSlugMap[ variant ]?.slug;
+		const version = botSlugMap[ variant ]?.version;
```

Comment:
I am a little worried that there is the possibility of these being `undefined` in future experiments and refactored code, perhaps we should protect against this?